netlify.toml

```diff
@@ -1,8 +1,3 @@
-[build.environment]
```

Comment:
Why this is removed , @sayantan1410 ?

Reply:
Ok, I see, @ydcjeff can you put this change to main and not in the PR ?

Reply:
It is already removed in main. https://github.com/pytorch-ignite/code-generator/commit/5954335fc5ef71e42da9ba81b24e74388c621e11